sdks/RepoToolset/tools/ProjectDefaults.props

```diff
@@ -132,4 +132,11 @@
     </When>
   </Choose>
 
+  <!-- 
+    Implements proposal https://github.com/dotnet/designs/pull/33.
+  -->
+  <ItemGroup Condition="'$(UsingToolNetFrameworkReferenceAssemblies)' == 'true'">
```

Comment:
I'd suggest also conditioning this on whether the `TargetFrameworkIdentifier` is `.NETFramework`.

Reply:
Wouldn't they be ignored if not? The build works with just this condition. Is there a potential problem?

Reply:
It's not a functional issue, but you'll end up with package references that aren't needed for projects that target something besides .NET Framework.  They won't actually do anything due to the logic inside them, but it seems cleaner not to reference them when you know they won't be needed.